astrology.lic

```diff
@@ -10,6 +10,12 @@ class Astrology
 
   def initialize
     settings = get_settings
+    arg_definitions = [
+	    [{name: 'train', regex: /train/i, description: 'Trains astrology'}],
```

Comment:
I would expect train to be the default behavior if no args are supplied. What do you think?